docs/configure-renovate-ce-github.md

```diff
@@ -63,6 +63,8 @@ Mend Renovate requires configuration via environment variables in addition to Re
 
 **`MEND_RNV_GITHUB_APP_KEY`**: A string representation of the private key of the provisioned Renovate app on GitHub. To insert the value directly into a Docker Compose environment variable, open the PEM file in a text editor and replace all new lines with "\n" so that the entire key is on one line. Alternatively, you can skip setting this key as an environment variable and instead mount it as a file to `/usr/src/app/renovate.private-key.pem`, as shown in the example Docker Compose file.
 
+**`RNV_GITHUB_PEM_FILE_PATH`**: The file path for GitHub app key. Defaults to `/usr/src/app/renovate.private-key.pem`.
```

Comment:
The sentence prior to this implies the path is fixed, so maybe it could be updated too

Reply:
update